src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -29,7 +29,12 @@ With just a few additions your existing Dockerfile can be used with our Node.js
 
    Install a specific version, or use any of the other options provided by the [`package.json` format](https://docs.npmjs.com/files/package.json#dependencies). Check [the Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes) for information about past agent versions.
 
-2. Depending on how your container is setup, you can edit the `ENTRYPOINT` to include `newrelic` module first with Node.js `-r`/`--require` flag by running `node -r newrelic YOUR_PROGRAM.js`. If you can't control how your program runs, you can load the `newrelic` module before any other module in your program by adding `require('newrelic')`.
+2. Depending on how your container is set up, you can edit the `ENTRYPOINT` to include `newrelic` module first with Node.js `-r`/`--require` flag by running ```node -r newrelic YOUR_PROGRAM.js```
+
+   or ```ENTRYPOINT ['node', '-r', 'newrelic', 'YOUR_PROGRAM.js']```
+
+   If you can't control how your program runs, you can load the `newrelic` module before any other module in your program by adding `require('newrelic')`.
+
 
    <Callout variant="tip">
```

Comment:
Same comment as before. we have 4 callouts in this doc. Let's choose one to prioritize and nest relevant information within procedure

Reply:
I've also noticed that the first callout in this doc mixed up _variant_ and _tip_. I'm in favor of one "tip," one "caution," and one "important."